docs/pages/world/reference/world.mdx

```diff
@@ -0,0 +1,461 @@
+[//]: # "This file is autogenerated, do not change manually"
+
+## World
+
+[Git Source](https://github.com/latticexyz/mud/blob/main/packages/world/src/World.sol)
+
+**Inherits:**
+[StoreData](/store/reference/store-core#storedata), [IWorldKernel](/world/reference/world-external#iworldkernel)
+
+_This contract is the core "World" contract containing various methods for
+data manipulation, system calls, and dynamic function selector handling._
+
+### State Variables
+
+#### creator
+
+Address of the contract's creator.
+
+```solidity
+address public immutable creator;
+```
+
+### Functions
+
+#### worldVersion
+
+```solidity
+function worldVersion() public pure returns (bytes32);
+```
+
+**Returns**
+
+| Name     | Type      | Description                                |
+| -------- | --------- | ------------------------------------------ |
+| `<none>` | `bytes32` | The current version of the world contract. |
+
+#### constructor
+
+_Event emitted when the World contract is created._
+
+```solidity
+constructor();
+```
+
+#### prohibitDirectCallback
+
+_Prevents the World contract from calling itself._
+
+```solidity
+modifier prohibitDirectCallback();
+```
+
+#### initialize
+
+Initializes the World by installing the core module.
+
+_Only the initial creator can initialize. This can be done only once._
+
+```solidity
+function initialize(IModule initModule) public prohibitDirectCallback;
+```
+
+**Parameters**
+
+| Name         | Type      | Description                                   |
+| ------------ | --------- | --------------------------------------------- |
+| `initModule` | `IModule` | The core module to initialize the World with. |
+
+#### installRootModule
+
+Installs a given root module in the World.
+
+_The caller must own the root namespace._
+
+```solidity
+function installRootModule(IModule module, bytes memory encodedArgs) public prohibitDirectCallback;
+```
+
+**Parameters**
+
+| Name          | Type      | Description                                        |
+| ------------- | --------- | -------------------------------------------------- |
+| `module`      | `IModule` | The module to be installed.                        |
+| `encodedArgs` | `bytes`   | The ABI encoded arguments for module installation. |
+
+#### \_installRootModule
+
+_Internal function to install a root module._
+
+```solidity
+function _installRootModule(IModule module, bytes memory encodedArgs) internal;
+```
+
+**Parameters**
+
+| Name          | Type      | Description                                        |
+| ------------- | --------- | -------------------------------------------------- |
+| `module`      | `IModule` | The module to be installed.                        |
+| `encodedArgs` | `bytes`   | The ABI encoded arguments for module installation. |
```

Comment:
can we filter this function out since it's an `internal` function? Probably has to be done in the post-processing